src/App.js

```diff
@@ -4,9 +4,10 @@ import MovieCard from './components/MovieCard'
 import Header from './components/Header'
 import MovieList from './components/MovieList'
 import SearchBar from './components/SearchBar'
+import RecentSearches from './components/RecentSearches'
 
 const DEFAULT_STATUS = 'Search a massive database of movies and TV shows!'
-const SearchStatus = ({text, color}) => <h1 className={color}>{text}</h1>
+const SearchStatus = ({text, color}) => <div className='dib'><h1 className={color}>{text}</h1></div>
```

Comment:
Now that there's a wrapper `div`, I would enclose in parentheses and add drop to a new line:

```
... => (
  <div>
    <h1></h1>
  </div>
)
```

Reply:
sounds good. done.